src/backend/launcher.ts

```diff
@@ -121,16 +124,32 @@ async function prepareLaunch(
     gameSettings.useSteamRuntime &&
     (isNative || gameSettings.wineVersion.type === 'proton')
   if (shouldUseRuntime) {
+    let nonNativeRuntime: SteamRuntime['type'] = 'soldier'
+    if (!isNative) {
+      try {
```

Comment:
A comment explaining what's being done here (and why it's done) might be helpful to future readers